tedana/utils.py

```diff
@@ -18,12 +18,14 @@
 from nilearn._utils import check_niimg
 from numpy import __version__ as numpy_version
 from pandas import __version__ as pandas_version
+from robustica import __version__ as robustica_version
 from scipy import __version__ as scipy_version
 from scipy import ndimage
 from scipy.special import lpmv
 from sklearn import __version__ as sklearn_version
 from sklearn.utils import check_array
 from threadpoolctl import __version__ as threadpoolctl_version
+from tqdm import __version__ as tqdm_version
```

Comment:
I don't think we're using tqdm. We should remove it from here or else users will get errors because it isn't a requirement in our `pyproject.toml` file.

Reply:
I think we should keep it as well.